### DISCUSSION TITLE
Trying to seed database with old backed-up data from project.

### DISCUSSION THREAD

**Discussion by samlroberts** [supabase/supabase#12830] [Feature Requests]
I needed to restore an old backup of our project db, and I wanted to be able to load that data into my local db using the cli, but encountered errors trying to restore the backup locally.

Would love the ability for the cli to be able to do this, would help with testing migrations against our staging data. Any thoughts?

**Comment by drewbaker** [comment]
I would also love an easy way to restore a backup of one project into another. Or an easy way to seed a branch with data from the main branch.
